deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -2,9 +2,17 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.namespace }}
+  labels: 
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
 
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.operator_namespace }}
+  labels:
+    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: privileged
```

Comment:
@anik120 @perdasilva 

This namespace has `pod-security.kubernetes.io/enforce: baseline` which means `privileged ` pods will not run. So, make no sense we add the ` pod-security.kubernetes.io/warn: privileged`